utils/zerovec/derive/Cargo.toml

```diff
@@ -32,3 +32,18 @@ serde = { workspace = true, features = ["derive"] }
 zerofrom = { path = "../../../utils/zerofrom" }
 bincode = { workspace = true }
 serde_json = { workspace = true }
+
+[[example]]
+name = "derives"
+harness = false
+test = true
+
+[[example]]
+name = "make"
+harness = false
+test = true
+
+[[example]]
+name = "make_var"
+harness = false
+test = true
```

Comment:
Seems like we need this only in `zerovec_derive`. It seems the examples in most other crates are already running in CI. Not clear what makes this crate special: maybe that it is a proc macro? And this is the only proc macro crate with examples that have assertions.

Reply:
yeah probably